### ISSUE TITLE
BGR image sent instead of RGB

### ISSUE THREAD

**Issue by guerwan** [google-ar/arcore-android-sdk#1556]
We are using ARFoundation on Unity and we are using the method:
public bool TryAcquireLatestCpuImage(out XRCpuImage cpuImage)

We are converting the result to RGBA32 as explained here:
https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@4.0/manual/cpu-camera-image.html
But on some devices we get are still getting a BGR image instead of RGB.

Versions used:
- Unity 2019.3.2f1
- ARFoundation 4.2.3
- Example of device on which this issue happens: Xiaomi M2007J20CG - Android OS 12 / API-31 SKQ1.211019.001/V13.0.3.0.SJGEUXM

**Comment by guerwan** [comment]
We also noted a similar issue on another Xiaomi device: 
- Xiaomi M2103K19PG - Android OS 11

**Comment by guerwan** [comment]
Any update on this issue?

**Comment by guerwan** [comment]
Also the same issue on Redmi Note 11 Pro 5G, Android version 11.
Any update would be very appreciated, thanks.

**Comment by ImportantCallFromMicrosoft** [comment]
Here is a code sample of how we implemented the conversion:
[colorConvSampleCode.txt](https://github.com/google-ar/arcore-android-sdk/files/11026399/colorConvSampleCode.txt)

**Comment by guerwan** [comment]
I agree it seems to be an issue happening only on some Xiaomi devices.
